reference/halyard/custom.md

```diff
@@ -173,12 +174,36 @@ Node selector annotations will put out `nodeSelector` values in the Pod specific
 
 ```
 kubernetes:
-  nodeSelectors:
-     exampleNodeKey: exampleNodeValue
+  deploymentEnvironment:
+    nodeSelectors:
+      exampleNodeKey: exampleNodeValue
 ```
 
 Additional information regarding Node Selectors can be found in [the Kubernetes NodeSelector documentation](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector).
 
+### affinity
+
+Affinity annotations will put out `affinity` values in the Pod specification and is more expressive than `nodeSelector`. Precedence can be set rather than a hard requirement. The two types are `node affinity` and `inter-pod affinity/anti-affinity`.
```

Comment:
```suggestion
Affinity annotations generate `affinity` values in the Pod specification and are more expressive than `nodeSelector`. You can set precedence rather than specify an affinity hard requirement. The two types are `node affinity` and `inter-pod affinity/anti-affinity`.
```

Reply:
```Affinity annotations may apply one or many `affinity` or `anti-affinity` rules to a Pod specification, allowing additional flexibility over `nodeSelector`.```